### PR TITLE
IL-10: Add button navigation to existing components

### PR DESCRIPTION
**Pull request by bnelson777**
-Clicking on messages in inbox now link to message_thread
-Message in inbox links to new message
-Clicking "add" in rolodex leads to scan
-Cleared message_thread and create_message links from home screen

Note: Some links not complete at time of pull request due to those screens not being available

### CODE REVIEW

**Comment by joewetton** [comment]
IOS 👍

**Review by joewetton — APPROVED** [review]
IOS 👍

**Review by hontatfai — APPROVED** [review]
Works on android emulator

File: app/components/home/index.js [hunk 1]
```diff
@@ -16,6 +16,10 @@ import * as ReduxActions from '../../actions'; //Import your actions
 import {Actions} from 'react-native-router-flux'
 
 class Home extends Component {
+    static navigationOptions = {
```
Comment (joewetton) [inline app/components/home/index.js:19]: Fixed bug where splash page to Home allowed you to go back to the splash page.

File: app/components/home/index.js [hunk 2]
```diff
@@ -65,14 +69,6 @@ class Home extends Component {
                         </View>
                     </TouchableHighlight>
 
-                    <TouchableHighlight onPress={() => {this.pressButton("Share")}}>
```
Comment (joewetton) [inline app/components/home/index.js:68]: Share will be from Wallet/rolodex till further notice

**Comment by bnelson777** [comment]
Thanks for fixing the conflicts Joe 👍

**Pull request merged**